docs/guide/hot-reload.md

```diff
@@ -42,3 +44,52 @@ if (module.hot) {
 ```
 
 Checkout the [counter-hot example](https://github.com/vuejs/vuex/tree/dev/examples/counter-hot) to play with hot-reload.
+
+## Dynamic module hot reloading
+
+If you use modules exclusively, you can use `require.context` to load and hot reload all modules dynamically.
+
+```js
+// store.js
+import Vue from 'vue';
```

Comment:
Could you remove all semicolons from the end of lines?

Reply:
Fixed in 0a25bfe.